doc/articles/getting-started/wizard/application.md

```diff
@@ -0,0 +1,3 @@
+Sets the iOS/macOS application [Bundle ID](https://developer.apple.com/documentation/appstoreconnectapi/bundle_ids) to be used in the App Store.
```

Comment:
Let's use header ID and xrefs for all pages we add: 
https://github.com/unoplatform/uno/blob/b862d99a1adb187830de662f35bf975ed6e611ef/doc/articles/Uno.UI.Toolkit.md?plain=1#L1-L3

Reply:
This page is meant to be included in another one and is not meant to be a standalone.
Does it still deserve its own title?

Reply:
Done